uportal-war/src/main/data/default_entities/permission_owner/UP_USERS.permission-owner.xml

```diff
@@ -30,6 +30,12 @@
         <targetProvider>userAttributesTargetProvider</targetProvider>
     </activity>
     <activity>
+        <name>View own attribute</name>
+        <fname>VIEW_OWN_USER_ATTRIBUTE</fname>
+        <desc>View the value(s) of one's own user attribute</desc>
```

Comment:
Suggest being explicit here that `VIEW_USER_ATTRIBUTE` on an attribute effectively implies this permission. Suggest also updating `desc` of `VIEW_USER_ATTRIBUTE` to note that it effectively implies this permission.